pages/02.Setup/03.Cron Jobs/docs.en.md

```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
 php /path/to/mautic/bin/console mautic:maintenance:cleanup --days-old=365 --dry-run
 ```
 
+### MaxMind CCPA complience
+
+MaxMind requires to keep a "do not sell" list up to date and remove all data that were added by those IP addresses in the past from MaxMind.
+
+See more details in the official MaxMind website: https://blog.maxmind.com/tag/ccpa/
```

Comment:
```suggestion
See more details in the official MaxMind website: [https://blog.maxmind.com/tag/ccpa/][maxmind-ccpa]
```

Reply:
Please add the link macro at the end of the page:

`[maxmind-ccpa]: <https://blog.maxmind.com/tag/ccpa/>`